server/data/quotes.json

```diff
@@ -27,6 +27,10 @@
     "sentence": "Learn a language, and you’ll avoid a war.",
     "author": "Arab Proverb"
   },
+  {
+    "sentence": "Those who know nothing of foreign languages know nothing of their own.",
```

Comment:
Hey @Alka-git12, thanks for opening this PR! However, this quote is a duplicate of lines  `147-148` of your file. 

Hope you can choose a different quote that is not already on the list.